hawc/apps/assessment/models.py

```diff
@@ -737,10 +740,41 @@ def try_read_df(cls, data, suffix: str, worksheet_name: str = None) -> pd.DataFr
         return df
 
 
+class Log(models.Model):
+    assessment = models.ForeignKey(
+        Assessment, blank=True, null=True, related_name="logs", on_delete=models.CASCADE
+    )
+    message = models.TextField()
+    created = models.DateTimeField(auto_now_add=True)
+    last_updated = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        ordering = ("-created",)
+
+
+class Blog(models.Model):
+    subject = models.CharField(max_length=128)
+    content = models.TextField()
+    rendered_content = models.TextField(editable=False)
+    published = models.BooleanField(default=False)
+    created = models.DateTimeField(auto_now_add=True)
+    last_updated = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        ordering = ("-created",)
+
+
+@receiver(pre_save, sender=Blog)
+def render_content(sender, instance, *args, **kwargs):
+    instance.rendered_content = to_html(instance.content)
```

Comment:
might want to catch an exception here and set to some value `<h1>Error - myst_parser Parsing error</h1>`, assuming the parser can crash